### DISCUSSION TITLE
Quitting out of Neogit?

### DISCUSSION THREAD

**Discussion by kvietcong** [NeogitOrg/neogit#290] [Q&A] [answered]
Sorry if this is a dumb question, but is there any way I can make escape quit out of Neogit? I tried making an autocommand on `BufEnter` and checking the buffer name to remap things but the callback never was called.

**Comment by TimUntersberger** [comment, marked as answer]
```lua
neogit.setup {
  mappings = {
    status = {
      ["<keybinding>"] = "Close"
    }
  }
}
```

**Reply by kvietcong** [reply]
🤦‍♂️ On further inspection of the config example I see it now. Sorry bout that and thank you.

Just another quick question. I was trying to use autocommands and hook into Neogit's BufEnter but it would never activate. It could just be my code but is there possibly a reason on Neogit's side?

**Reply by kvietcong** [reply]
Also is there a way to apply this to all Neogit windows like the help popup and etc? I tried combin the documentation and code and couldn't find the specific mode options for it
